UUSwift/UURemoteData.swift

```diff
@@ -87,6 +87,10 @@ public class UURemoteData : NSObject, UURemoteDataProtocol
         {
             return data
         }
+		
+		if remoteLoadCompletion == nil {
+			return nil
+		}
```

Comment:
I'm not sure we want this.  There is still the NSNotification that fires when a download is complete, so this would disable using only that.  Or if we truly want to deprecate the NSNotification style, we can just remove it.

Reply:
Good catch.